### PR TITLE
BG-13088 / Update trade payload to v1.1.1

### PR DESCRIPTION
**Pull request by dacruz21**
Updating the trade payload to match schema 1.1.1. Also added a version field. Payload is still built locally.

@tylerlevine this constitutes a breaking change to the `buildPayload` and `buildAndSignPayload` functions. Is there a `next` branch I should merge this into so it goes out with the next major version bump? The current `next` branch is a few months out of date but I don't want to delete it as there are unmerged commits.

### CODE REVIEW

**Comment by tylerlevine** [comment]
Are we going to need this change soon? I wasn't planning on doing another major version in the short term.

File: modules/core/src/v2/trading/payload.ts [hunk 1]
```diff
@@ -2,10 +2,11 @@
  * @prettier
  */
 export interface Payload {
-  walletId: string;
+  version: '1.1.1';
```
Comment (quazeenii) [inline modules/core/src/v2/trading/payload.ts]: maybe consider making this a config somewhere?

**Comment by tylerlevine** [comment]
@dacruz21 is this on ice while you guys figure out some stuff?

**Comment by dacruz21** [comment]
Yep, I'm waiting on the payload change on the backend to go through then I'll finish up this PR. Feel free to close if you want

File: modules/core/test/v2/unit/trading/payload.ts [hunk 2]
```diff
@@ -69,5 +111,54 @@ describe('Trade Payloads', function() {
...
+            {
+              accountId: 'test_counterparty_2',
+              currency: 'ofctusd',
+              amount: '0'
+            }
+          ]
+        })
+      });
+
+    const error = yield bitgo.getAsyncError(tradingAccount.buildPayload({
```
[the remark this answers is not in the document]
Comment (dacruz21) [reply modules/core/test/v2/unit/trading/payload.ts]: oh sweet

**Pull request merged**